algorithms/heap/binary_heap.py

```diff
@@ -70,26 +81,26 @@ def perc_up(self, i):
         perc_up the minimum element
         Complexity: O(logN)
     """
+
     def insert(self, val):
+        """Inserts the given value to the heap"""
```

Comment:
Could we use the existing doc-string (that happens to be placed above instead of inside the function)? Same thing for other functions below ↓

Reply:
fixed